codecept.conf.js

```diff
@@ -77,6 +77,14 @@ exports.config = {
           mochaFile: 'test-results/result.xml',
         },
       },
+      'mochawesome': {
+        stdout: './output/console.log',
```

Comment:
This is absolutely useless because it reports output from one chunk only. Replace with `stdout: '-',`.

Reply:
Replaced 'stdout: './output/console.log' with stdout: '-', as per the comments Damian.